File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.py
class Solution(object):
    def maxProduct(self, n):
        """
        :type n: int
        :rtype: int
        """
        # n = [int(d) for d in str(n)]
        # n.sort()
        m = sorted(str(n))
        return int(m[-1]) * int(m[-2])
        

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cs
public class Solution {
    public int MaxProduct(int n) {
        var sorted = n.ToString().OrderBy(c => c).ToArray();
        int len =sorted.Length;
        int n1 = sorted[len - 1] - '0';
        int n2 = sorted[len - 2] - '0';
        return n1 * n2;

    }
}